test/rebar_hooks_SUITE.erl

```diff
@@ -98,6 +100,17 @@ run_hooks_once(Config) ->
     rebar_test_utils:create_app(AppDir, Name, Vsn, [kernel, stdlib]),
     rebar_test_utils:run_and_check(Config, RebarConfig, ["compile"], {ok, [{app, Name, valid}]}).
 
+%% test that even if a hook is defined in a used profile it is not run twice
```

Comment:
The discussion on IRC mentioned things that looked like umbrella apps, but here it doesn't seem to matter?